app/main.py

```diff
@@ -119,6 +120,18 @@ async def get_forecast_summaries(model: ModelEnum, request: schemas.StationCodeL
         LOGGER.critical(exception, exc_info=True)
         raise
 
+@app.post('/noon_forecasts/', response_model=schemas.NoonForecastResponse)
+def get_noon_forecasts(request: schemas.StationCodeList, _: bool = Depends(authenticate)):
+    """ Returns noon forecasts pulled from BC FireWeather Phase 1 website for the specified
+    set of weather stations. """
+    try:
+        start_date = datetime.datetime.now().replace(hour=0, minute=0, second=0)
```

Comment:
Please throw in a LOGGER.info('/noon_forecasts/') right after the try - I'm trying to have that on every api call.